src/main/java/org/sagebionetworks/web/client/Portal.java

```diff
@@ -161,50 +167,57 @@ private void continueInit() {
-                                    ginjector.getViewDefaultColumns();
+                                // start version timer
+                                ginjector.getVersionTimer().start();
+                                // start timer to check for Synapse outage or scheduled maintenance
+                                ginjector.getSynapseStatusDetector().start();
+                                // Goes to place represented on URL or default place
+                                historyHandler.handleCurrentHistory();
+                                globalApplicationState.initializeDropZone();
+                                globalApplicationState.initializeToastContainer();
+                                // initialize the view default columns so that they're ready when we need them (do this by constructing that singleton object)
+                                ginjector.getViewDefaultColumns();
 
-                                    // start timer to check for user session state change (session expired, or user explicitly logged
-                                    // out).  Backend endpoints must be set before starting this (because it attempts to get "my user profile")
-                                    ginjector.getSessionDetector().start();
+                                // start timer to check for user session state change (session expired, or user explicitly logged
+                                // out).  Backend endpoints must be set before starting this (because it attempts to get "my user profile")
+                                ginjector.getSessionDetector().start();
 
-                                    // start a timer to check to see if we're approaching the max allowable space in the web storage.
-                                    // clears out the web storage (cache) if this is the case.
-                                    ginjector
-                                      .getWebStorageMaxSizeDetector()
-                                      .start();
-                                  }
-                                }
-                              );
-                            });
-                        });
-                    }
-                  }
+                                // start a timer to check to see if we're approaching the max allowable space in the web storage.
+                                // clears out the web storage (cache) if this is the case.
+                                ginjector
+                                  .getWebStorageMaxSizeDetector()
+                                  .start();
+                              }
+                            }
+                          );
+                        } catch (Exception e1) {
+                          // TODO Auto-generated catch block
+                          e1.printStackTrace();
```

Comment:
Should this `try`/`catch` only wrap the feature flag-related logic?

Reply:
Yes and yes.  Updated the code